openid-data-classes/src/commonMain/kotlin/at/asitplus/openid/AuthenticationRequestParameters.kt

```diff
@@ -267,4 +351,23 @@ data class AuthenticationRequestParameters(
             jsonSerializer.decodeFromString<AuthenticationRequestParameters>(it)
         }.wrap()
     }
+}
+
+/**
+ * CSC: Multiple hash values can be passed as comma separated values,
+ * e.g. oauth2/authorize?hash=dnN3ZX.. .ZmRm,ZjIxM3… Z2Zk,…
+ */
+object HashesSerializer : KSerializer<List<ByteArray>> {
+    override val descriptor: SerialDescriptor =
+        PrimitiveSerialDescriptor("HashesSerializer", PrimitiveKind.STRING)
+
+    override fun deserialize(decoder: Decoder): List<ByteArray> {
+        val listOfHashes = decoder.decodeString().split(",")
+        return listOfHashes.map { at.asitplus.dif.jsonSerializer.decodeFromString(ByteArrayBase64Serializer, it) }
```

Comment:
but the documentation for `hashes` says it's base64-URL encoded

Reply:
According to CSC-API v2.0.0.2 hashes are base64 encoded. Just double checked

Reply:
If this is inconsistent with the RQes spec then I think I prefer the CSC version as it is likely to change to CSC soon